.github/workflows/flutter_integration_test.yml

```diff
@@ -0,0 +1,73 @@
+name: flutter integration tests
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
```

Comment:
Let's add the `styfle/cancel-workflow-action` job as well (See other workflows).